cli/src/bin/phylum.rs

```diff
@@ -146,7 +130,9 @@ async fn handle_commands() -> CommandResult {
         let res = update::do_update(matches.is_present("prerelease")).await;
         spinner.stop();
         println!();
-        return res.map(CommandValue::String);
+        let message = res?;
+        print_user_success!("{}", message);
```

Comment:
You are expertly dodging land mines here.... I definitely would have written this as:

```
print_user_success!("{}", res?);
```

But now I realize that thanks to our definition of `print_user_success!`, that would print the ✅ even on failure... 🤦

Reply:
Yeah, I initially considered doing this, but then thought that's just a bug waiting to happen regardless of the current `print_user_success` implementation. Better to be explicit and avoid any potential for misinterpretation.